### ISSUE TITLE
Map Notes in Map Check - QOL enhancement

### ISSUE THREAD

**Issue by SirNorek** [SnosMe/awakened-poe-trade#440]
When using map mod check I noticed that some maps have a picture of the boss (sidenote: would love if all maps did) but it got me thinking, I'd also love to be able add notes there based on the map! There are a ton of maps and remembering a million details every league is not my strong point, but it would be great if I could add whatever I felt was important to know.

ex: good card drop, or I really hate that boss, or a reminder of what boss is on that map etc.

Probably not great examples, but I'm sure you see the potential.

Thanks for all the work done to make APT the beast is it 👍

**Comment by SnosMe** [comment]
There's an inconvenience I see: you don't have anything to note until you've entered the map. But once you have entered it and great idea come in your head, you now don't have a map item to press hotkey on it.

**Comment by SirNorek** [comment]
> There's an inconvenience I see: you don't have anything to note until you've entered the map. But once you have entered it and great idea come in your head, you now don't have a map item to press hotkey on it.

You're totally right, would have to add a searchable map list similar to the map mods list we use for map checking, but it would then have access to our notes in another/extended frame. Either way I thought it would be rather useful, so threw the idea into the hat ;)

**Comment by flapanther33781** [comment]
> There's an inconvenience I see: you don't have anything to note until you've entered the map. But once you have entered it and great idea come in your head, you now don't have a map item to press hotkey on it.

Not necessarily true.  If we encounter a map in our stash that has no note we could go to Youtube and watch a walkthrough of that map, remind ourselves of what we'd want to put in the note, and then do so.  But if a user fails to do so, or chooses not to, is that your problem?  Or anyone else's?  I don't think so.

**Comment by icesharkk** [comment]
I have been wanting this for a long time. even if you added an organized folder structure in appdata that I could update with notes and pictures of my own would amazing. I really want to make little minimalist map layout pictures to attach to each of these along with a rating for the map. I'm using the widget system for now but there is no cross compatibility with the map analyze function.  

I'm glad you were already look at this function though. I rely heavily on the map analysis function since I am to old to memorize all of the details and strategy anymore. the more I could configure and update map info as the league progresses the better.

**Comment by Igonorant** [comment]
I really love the map checking feature, but I also have some suggestions :)

- Instead of a picture of the boss, maybe a layout like [these ones](https://docs.google.com/document/d/1sExA-AnTbroJ-HN2neZiij5G4X9u2ENlC7m_zf1tqP8/edit#) would be very helpfull.
- Adding some extra information like [these ones](https://docs.google.com/spreadsheets/d/1fIs8sdvgZG7iVouPdtFkbRx5kv55_xVja8l19yubyRU/edit#gid=202858850) (Natural Tier, Layout, Mob Density, Div Cards, Boss) also would be very nice!
